src/algorithms/tonal/audio2pitch.cpp

```diff
@@ -0,0 +1,130 @@
+#include "audio2pitch.h"
+
+using namespace essentia;
+using namespace standard;
+
+const char* Audio2Pitch::name = "Audio2Pitch";
+const char* Audio2Pitch::category = "Pitch";
+const char* Audio2Pitch::description = DOC("Extractor algorithm to compute pitch with several possible pitch algorithms, specifically targeted for real-time pitch detection on saxophone signals.");
```

Comment:
Currently, in all algorithms this DOC string starts with a sentence "This algorithm ..." summarizing what the algorithm does, which is then auto-parsed by doc generation script to create short descriptions for the [algorithm index](https://essentia.upf.edu/algorithms_reference.html). Let's keep it similar.